pilot/cmd/pilot-discovery/app/options.go

```diff
@@ -41,6 +42,19 @@ func secureTLSCipherNames() []string {
 	return sets.SortedList(cipherKeys)
 }
 
+// validateClusterAliases validates that there's no more then alias per clusterID.
+func validateClusterAliases(clusterAliases map[string]string) error {
+	seenClusterIDs := make(map[string]struct{})
```

Comment:
nit: Can you set instead of `map[string]struct{}`

Reply:
Yep, definitely makes sense. Didn't realise that Istio have its own sets helper package.